src/core/core.js

```diff
@@ -393,6 +393,10 @@ export default class Core {
         querySource: this.storage.get(StorageKeys.QUERY_SOURCE),
         additionalHttpHeaders: this._additionalHttpHeaders
       })
+      .then(response => {
+        console.log(response);
+        return response;
+      })
```

Comment:
I assume this was for debugging - forgot to delete?

Reply:
Whoops! Yep, forgot to delete.